store/driver/tikv_driver.go

```diff
@@ -331,3 +334,26 @@ func (s *tikvStore) CurrentVersion(txnScope string) (kv.Version, error) {
 func (s *tikvStore) ShowStatus(ctx context.Context, key string) (interface{}, error) {
 	return nil, kv.ErrNotImplemented
 }
+
+// GetLockWaits get return lock waits info
+func (s *tikvStore) GetLockWaits() ([]*deadlockPB.WaitForEntry, error) {
+	stores := s.GetRegionCache().GetStoresByType(tikvrpc.TiKV)
+	var result []*deadlockPB.WaitForEntry
+	for _, store := range stores {
+		// todo: try RegionCache.getStoreAddr with backoff
```

Comment:
I think it's not necessary because `GetStoresByType` only returns `resolved` stores.